packages/client/src/Dashboard.tsx

```diff
@@ -120,14 +121,23 @@ const useStyles = makeStyles(theme => ({
 export default function Dashboard({ accountId, navigate }: Props) {
   const classes = useStyles()
   const [open, setOpen] = React.useState(false)
-  const [isSearching, setIsSearching] = React.useState(false)
-  const [searchQuery, setSearchQuery] = React.useState("")
+  // const [isSearching, setIsSearching] = React.useState(false)
+
+  const [query, setQuery] = useQueryParams({
+    searchQuery: StringParam,
+    isSearching: BooleanParam
+  })
+  const { searchQuery, isSearching } = query
+
+  // const [searchQuery, setSearchQuery] = React.useState("")
   const getAccountResult = graphql.useGetAccountQuery({
     variables: { accountId: accountId! }
   })
-  const skipSearch = !isSearching || searchQuery.length < 3
+  const skipSearch =
+    !isSearching || (searchQuery ? searchQuery.length < 3 : false)
```

Comment:
The ternary could be shortened to `!!searchQuery && searchQuery.length < 3`. But that is a matter of taste - it is not necessarily better.

But as I said above it would be cleaner to assign `searchQuery` so that it is always a `string`.